lib/matplotlib/backends/backend_qtagg.py

```diff
@@ -47,25 +46,19 @@ def paintEvent(self, event):
             right = left + width
             # create a buffer using the image bounding box
             bbox = Bbox([[left, bottom], [right, top]])
-            buf = memoryview(self.copy_from_bbox(bbox))
```

Comment:
I realised that this PR should also cover qtcairo, which has virtually the same implementation (but uses the pre-multiplied format)